ui/i18n/messages.json

```diff
@@ -278,6 +280,7 @@
   "shareButton.operation.copied": "Link Copied",
   "shareButton.operation.copy": "Copy Link",
   "shareButton.tooltip": "Share",
+  "parentInspectorButton.tooltip": "Parent Inspector",
```

Comment:
Where is this label being used? It it is not used in nuxeo elements but only on nuxeo web ui, it should reside in the nuxeo web ui repository.